Jenkinsfile

```diff
@@ -1 +1,6 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 17],
+    [platform: 'windows', jdk: 11],
```

Comment:
as mentioned before - this is not something I believe is good.  it saves some compute $$$ but does not save any wall clock time on the build and makes windows related issues more of "is this windows or jdk" in a non obvious way (esp when the tests pass locally due to some timing thing..

Reply:
This `Jenkinsfile` is currently the stock recommendation. If you wish to suggest a different stock `Jenkinsfile`, run it by ci.jenkins.io admins and propose that change in `archetypes`.